wayback-core/src/main/java/org/archive/wayback/util/webapp/PortMapper.java

```diff
@@ -138,6 +138,22 @@ public RequestHandlerContext getRequestHandlerContext(
 		} else {
 			LOGGER.finer("No mapping for " + key);
 		}
+
```

Comment:
Does line 133 (`handler = pathMap.get(key);`) return `null` for paths with `;` in the middle? It may be better to wrap the `paramSep` work in an `if...else` block with lines 133 to 140.

Reply:
Ah ok, I just added new code after the existing... it's silly to have  more lines after `else`!